### PR TITLE
Implement Timify quote shape

### PR DESCRIPTION
**Pull request by locks**
Closes https://github.com/simplabs/simplabs.github.io/issues/883.

### CODE REVIEW

**Comment by locks** [comment]
Needs new image.

**Comment by ghislaineguerin** [comment]
Here's the image file. Can you add it to the PR and then I can adjust cropping?
image
Abstract Link:
https://share.goabstract.com/9ed22beb-d90c-414c-863e-67efd28b16a4?mode=design&sha=c428d39bd14a3de12f9a173409e5a9b19c14c47f

**Comment by locks** [comment]
Here's how it currently looks:

image

**Comment by ghislaineguerin** [comment]
@locks looks pretty good to me.

**Review by marcoow — CHANGES_REQUESTED** [review]
The image is 4.5MB 🐘

File: src/ui/components/ShapeQuoteTimify/template.hbs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<ShapeFeature @srcset="/assets/images/work/timify-quote.png">
+  <p>
+    <q typography:class="quote" lang="en">
+      simplabs' experienced engineers delivered a solid and well architected foundation for our web app. They also helped us establish best practices and a lean process internally. Working with them was a pleasure.
+    </q>
+  </p>
+  <p typography:class="small">
+    ANDREAS KNÜRR, TIMIFY CEO
```
Comment (marcoow) [inline src/ui/components/ShapeQuoteTimify/template.hbs]: Wouldn't it be better to do the up-casing via CSS?

Comment (locks) [reply src/ui/components/ShapeQuoteTimify/template.hbs]: `small` already does it, HTML corrected!

**Comment by locks** [comment]
@marcoow image shrunken to 600x and compressed to jpg in accordance to similar images that do not require alpha channel.

File: src/ui/components/ShapeQuoteTimify/template.hbs [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<ShapeFeature @srcset="/assets/images/work/timify-quote.jpg">
```
Comment (marcoow) [inline src/ui/components/ShapeQuoteTimify/template.hbs]: This is not actually a prope `srcset` – see https://github.com/simplabs/simplabs.github.io/blob/9fde3763e97c2c65ca2348214b9937a888ad3d07/src/ui/components/ShapeFeatureTimify/template.hbs for an example

**Review by marcoow — APPROVED** [review]

**Comment by marcoow** [comment]
I compressed the images. I recommend always running https://imageoptim.com/mac on all images.

**Comment by locks** [comment]
I thought I did, maybe I missed the final jpg's 🤔 Thank you!

**Pull request merged**